recipe/brand_python.py

```diff
@@ -35,7 +35,7 @@ def patch_get_version(msg):
 
 msg = os.environ.get('python_branding', '<undefined>')
 if msg == '<undefined>':
-    msg = "| packaged by conda-forge |" 
+    msg = "| packaged by Anaconda, Inc. |" 
```

Comment:
@alexlang74 - Should we have IBM here?

Reply:
@alexlang74 - Please let me know about this.

Reply:
Yes, please add IBM